packages/odyssey/src/scss/components/_checkbox.scss

```diff
@@ -14,15 +14,26 @@
 
 /* stylelint-disable no-descending-specificity, max-nesting-depth */
 
+$size-checkbox: 1.1428571429em;
```

Comment:
We can remove this and use one of our type-relative spacing units now, right? (similar to Odyssey's current implementation)